tsrc/repo.py

```diff
@@ -17,6 +17,19 @@ class Copy:
     dest = attr.ib()  # type: str
 
 
+@attr.s(frozen=True)
+class Link:
+    src = attr.ib()  # type: str
+    tgt = attr.ib()  # type: str
+
+
+@attr.s(frozen=True)
```

Comment:
We don't need this class right now

Reply:
I assume you mean the CopyDir class?  The highlighted in the review didn't make that clear.

Yes, just getting a bit ahead of myself.